subprojects/gst-plugins-bad/sys/msdk/gstmsdkcontext.c

```diff
@@ -590,8 +591,6 @@ gst_msdk_context_add_alloc_response (GstMsdkContext * context,
 {
   context->priv->cached_alloc_responses =
       g_list_prepend (context->priv->cached_alloc_responses, resp);
-
-  create_surfaces (context, resp);
```

Comment:
After delete this,  the create_surfaces() function seems no usage, please delete. Or the compiler will generate warning.